test/test_files/sbol3_genbank_conversion/test_extra_properties_with_references.gb

```diff
@@ -1,15 +1,15 @@
-LOCUS       AF165912                5485 bp    DNA     linear   PLN 29-JUL-1999
+LOCUS       AF165912                5485 bp    DNA     circular PLN 29-JUL-1999
```

Comment:
This accession isn't circular. Is this an intentional chance for testing purposes?

Reply:
I actually changed the original value here, as I guess the default value assigned is "linear" but I wanted to see if the accessions are actually getting set from our stored SBOL properties or are just the default ones.

Reply:
That's fine; I would just suggest adding a note into the description field to make it clear that this isn't the original AF165912 GenBank file